### PR TITLE
README update

### PR DESCRIPTION
**Pull request by CJ-Rose**
- remove unused, add new tech and its role to stack section
- edit copy, add links to IPFS and ERC-1155 in 'Data Models' section
- re-organize 'Getting Started' section to include instructions to run locally
- update 'Prerequisites' for Next and Sepolia network config
- add introductory info and docs links for all tech to 'Helpful References'

### CODE REVIEW

**Comment by CJ-Rose** [comment]
I've added intro descriptions of all the tool used in the repo, thinking it would be helpful for incoming contributors (including less-technical) to get an overview of all the moving parts. Might be too much, idk, what does everyone think?

'Server Design' section needs to be fleshed out more.. currently only `fetchReports` function is discussed.

Would 'Data Models' section be a good place to add a few more definitions of concepts, like 'Impact Funding', 'BC ratio'? or better to leave those to the FAQ page?

File: README.md [hunk 1]
```diff
@@ -74,38 +63,57 @@ The `/impact-reports` endpoint is responsible for serving impact reports. The im
...
-- [Vite Documentation](https://vitejs.dev/)
-- [Remix-Auth Documentation](https://github.com/sergiodxa/remix-auth)
-- [WalletConnect Documentation](https://docs.walletconnect.com/)
-- [Alchemy Documentation](https://docs.alchemy.com/)
+- [TypeScript Documentation](https://www.typescriptlang.org/docs/) TypeScript is a strongly typed programming language that builds on JavaScript, giving you better tooling at any scale.
+- [Next.js Documentation](https://next.org/docs/) Next.js is a React framework for building full-stack web applications. You use React Components to build user interfaces, and Next.js for additional features and optimizations.
+Under the hood, Next.js also abstracts and automatically configures tooling needed for React, like bundling, compiling, and more. This allows focus on building the application instead of spending time with configuration.
+- [React Documentation](https://react.dev/reference/react/) React apps are made out of components. A component is a piece of the UI (user interface) that has its own logic and appearance. A component can be as small as a button, or as large as an entire page. React components are JavaScript functions that return markup.
```
Comment (thebeyondr) [inline README.md:110]: dont think this is needed since Nextjs implies React

File: README.md [hunk 2]
```diff
@@ -52,20 +47,14 @@ We recommend [direnv](https://direnv.net/) for managing your environment variabl
...
 
 - `fetchReports`: Function to retrieve reports, including interaction with Hypercerts.
 
-### Data Models
-
-- **Impact Report**: The report or stories that have been published previously and verified to actually produce an impact.
-- **Hypercert**: A token representing a claim of impactful work, which is fractionable and transferable, conforming to the ERC-1155 standard for semi-fungible tokens.
-- **Hypercert Metadata**: A set of data associated with a Hypercert, detailing the scope of work, contributors, impact, and rights, stored on IPFS.
-
 ### Separation of Concerns
```
Comment (thebeyondr) [inline README.md:58]: might be getting too deep with this

**Review by thebeyondr — CHANGES_REQUESTED** [review]
lets strip this back to get a dev up and running so they can read the code for themselves. I think there's a lot of stuff that helpful in here though, thank you!

File: README.md (hunk 2 above)
Comment (CJ-Rose) [reply README.md:58]: ok so you want to remove
- both references to React
- entire "Data Models" section
- all of "Server Design" section, or leave as it but no additional content?
- keep the expanded content in "Helpful References"

correct or no?

Comment (thebeyondr) [reply README.md:58]: yeah, you can remove them imo. the deeper we get the more we'll have to change if the tech changes

**Review by thebeyondr — APPROVED** [review]

**Pull request merged**